traveler-ios-ui/TravelerKitUI/ViewControllers/ParkingResultMapListController.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  ParkingQueryViewController.swift
+//  ParkingResultMapListViewController.swift
```

Comment:
`ParkingResultMapViewController` makes more sense

Reply:
This is the container of both ParkingResultMapViewController and ParkingResultListViewController.